src/reference-designs/tko-on-vsphere-nsx.md

```diff
@@ -320,11 +320,11 @@ We propose deploying Tanzu Kubernetes Grid on non-stretched vSphere clusters. Al
 
 ## Container Registry
 
-The Tanzu Editions include [Harbor](https://goharbor.io/) as a container registry. Harbor provides a location for pushing, pulling, storing, and scanning container images used in your Kubernetes clusters. There are three main supported installation methods for Harbor:
+The Tanzu Editions include [Harbor](https://goharbor.io/) as a container registry. Harbor provides a location for pushing, pulling, storing, and scanning container images used in your Kubernetes clusters. There are three officially documented installation methods for Harbor:
```

Comment:
You can install Harbor using the following methods:

Reply:
This should be addressed in the latest push